File: Day27/DSA/Number_of_senior_citizens.cpp
Code:- 

class Solution {
public:
    int countSeniors(vector<string>& details) {
        int age = 0;
        for (auto i : details) {
            if (stoi(i.substr(11, 2)) > 60)
                age++;
        }
        return age;
    }
};

File: Day27/DSA/Number_of_Senior_Citizens.java
Question:-  https://leetcode.com/problems/number-of-senior-citizens/description/

Code:- 

class Solution {
    public int countSeniors(String[] details) {
        int count  = 0;

        for(int i = 0;i<details.length;i++)
        {
            String age = details[i].substring(11,13);
            System.out.println(age);
            if(Integer.parseInt(age) > 60)
            {
                count++;
            }
        }

        return count;
    }
}